<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Frame_query</class>
 <widget class="QWidget" name="Frame_query">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>366</width>
    <height>291</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="query_by_course">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>本学期每次课出勤概览</string>
   </property>
  </widget>
  <widget class="QTableView" name="courseSign">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>171</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableView" name="signInfo">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>50</y>
     <width>171</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="signedNum">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>10</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>签到人数:</string>
   </property>
  </widget>
  <widget class="QLabel" name="unSignedNum">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>30</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>未签到人数:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
